spec/features/schools/register_an_ect/previously_registered_but_inactive_spec.rb

```diff
@@ -0,0 +1,116 @@
+RSpec.describe 'Registering an ECT' do
+  include_context 'fake trs api client'
+
+  scenario 'previously registered but inactive' do
+    given_i_am_logged_in_as_a_state_funded_school_user_who_has_previously_registered_an_ect
+    and_i_am_on_the_schools_ects_index_page
+    and_i_start_adding_an_ect
+    and_i_click_continue
+    and_i_submit_the_find_ect_form
+    and_i_choose_that_the_details_are_correct
+    and_i_click_confirm_and_continue
+    then_i_am_on_the_view_previous_choices_page
```

Comment:
to prevent confusion, can we rename this step to something like: `then_i_am_on_the_registered_before_page`

Reply:
Good shout - done